src/eterna/rscript/RScriptEnv.ts

```diff
@@ -215,8 +215,6 @@ export default class RScriptEnv extends ContainerObject {
 
     public getUIElement(type: RScriptUIElementID, i: number = -1): RScriptUIElement | null {
         switch (type) {
-            case RScriptUIElementID.ACTION_MENU:
-                return this.ui.toolbar.actionMenu;
```

Comment:
We'll need to audit the tutorial script for puzzle 6502961. It looks like it just uses this reference in a ShowUI which should be a noop and be safe to remove from the tutorial script, but need to confirm.